FraihmworkHealthAndIntegrityMonitoringApiChangelist.txt

```diff
@@ -1,3 +1,6 @@
+Version 0.0.10:

```

Comment:
Future merge conflicts, HO!

In retrospect, maybe we should have included both of these as a single change for the external api repository, since, to the end user, it is only one change